### PR TITLE
GLVisualize JSoC post

### PR DESCRIPTION
**Pull request by SimonDanisch**
Hi all,
hope this is good as it is. Sorry for eventual errors, it got pretty late :( Feel free to make corrective changes!
Thanks for making JSoC possible!

@Keno, @shashi, @jiahao, @ViralBShah 

Best,
Simon

### CODE REVIEW

File: blog/_posts/2015-10-22-glvisualize.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+To give you a short intro: GLVisualize is an interactive visualization library, which supports 2D and 3D rendering and building of basic GUIs. It's written entirely in Julia and OpenGL.
+
+One of the most interesting parts of GLVisualize is, that it's combining GUIs and visualizations, instead of relying on some library like QT and then doing the visualizations seperated from it.
+The reason is, that interactive visualization share a lot of infrastructure also needed for GUIs.
+By combining these two, new features are possible, e.g. text editing of labels in 3D space, or making elements of a visualization work like a button.
+The downside obviously is, that it takes a lot of work to get everything working nicely.
+This is why I'm especially thankful, that I was able to get funding via Julia Summer of Code to continue my work on this project.
+
+What have I been doing during Julia summer of code?
+A surprisingly large amount of time went into improving together with [Tim Holy](https://github.com/timholy).
```
Comment (jakebolewski) [inline blog/_posts/2015-10-22-glvisualize.md]: You need to say FileIO here

**Comment by SimonDanisch** [comment]
Forgot to include @dpsanders @timholy. Hope it is okay to use your material!

**Comment by timholy** [comment]
Naturally. Looks like a nice post!

**Comment by ViralBShah** [comment]
Is this ready to go? Do squash.

**Comment by dpsanders** [comment]
Of course, no problem!

**Comment by SimonDanisch** [comment]
Okay, squashed! This should be ready if no one complains.

**Comment by shashi** [comment]
:+1:

**Comment by dpsanders** [comment]
Looks really great Simon, congrats! The billiard demo doesn't seem to be working though.

**Comment by dpsanders** [comment]
I meant the gif, I haven't looked at the code yet.

**Comment by SimonDanisch** [comment]
Okay, code wouldn't have been a big surprise! it works when I open the `.md` on github directly but not on the julia site. Maybe the missing `?raw=true` is doing this (still surprising that it was working most of the time).

**Pull request merged**